### DISCUSSION TITLE
Manual Bed Leveling w/BLT Idea

### DISCUSSION THREAD

**Discussion by smooreace** [Jyers/Marlin#426] [Ideas]
Is it possible to create a manual bed leveling process that uses a BL Touch as "paper"? Here's the thought...

So you start with an ABL process that ends with a Home
From there the probe is moved to the corner that has to closest to the median ABL variation (assuming a 4 point bed leveling model like the Ender series printers). 
Example being 0,0 is -0.1, 0,235 is -.032 235,0 is 0 and 235,235 is +.01... average variation is -.0305, so 0,235 is the starting corner

Probe is moved to that corner and set a Z that would trigger the probe when the bed hits 0. Corner is adjusted until the probe triggers
Probe is moved to the opposite corner and maintains that Z setting. Corner is adjusted until the probe triggers

Rinse and repeat for the other 2 corners

Auto bed leveling is again triggered and the above adjustment cycle is processed. 

The end result is a bed VERY close to ZERO Z on all corners.. the optimum tramming situation. Then the Bed level scan can be done at whatever frequency to get a completely accurate topographical map of the bed, at a near-zero state.

**Comment by Cal-LF** [comment]
There was also mention of having an offset for using a dial indicator.

**Comment by HeMaN-NL** [comment]
+1

I had the same idea to use a BLT for manual bed leveling. Came to the idea section to submit it and find it already there.

I was more thinking in the way of doing an automatic bed level to retrieve the current values/offsets, and then using the BLT in a manual bed level action to read the Z-position, adjust the bed, read z-position again, just like you do now with a paper only use the BLT for more acurate measuring.

**Comment by Reggi3** [comment]
I think the feature already exists, it just needs enabling and cleaned up if it causes issues...
https://github.com/Jyers/Marlin/blob/Ender_3_V2_Extensible_UI/Marlin/Configuration.h#L1535
`#define LEVEL_CORNERS_USE_PROBE`

There's also an octoprint plugin that does this without needing to edit the configs and re-compile:
https://github.com/j-be/AutoBim

**Reply by HeMaN-NL** [reply]
hmm, think I need to do some research to compile my own version.
if I interpret lines 1543 and 1544 right it should be there already indeed. Thnx!

**Reply by HeMaN-NL** [reply]
I compiled a new fw with those options enabled, but there was no menu entry to use this functionality.
[posted a question in the Q&A](https://github.com/Jyers/Marlin/discussions/865#discussioncomment-760097) and got the reply that it is currently not implemented in the Ender3 v2 DWIN screens. 

Functionality seems to be existing, bot not reachable yet via the UI.

Hope this menu item will make it in one of the upcoming releases ;)
